### ISSUE TITLE
Server crash on fast movement v3.6.3

### ISSUE THREAD

**Issue by SlimeDog** [konsolas/AAC-Issues#1522]
Spigot 1.13.2 latest
ProtocolLib 4.4.0-b422 
AAC 3.6.1
ServerTutorialPlus 1.20.0

NOTE: Repot submitted to spigot.org as well.

I am running a ServerTutorialPlus tutorial in creative mode that transports the player to distant unloaded chunks, after a block click at each point. Without AAC protections, it works fine. The AAC configuration is nearly-default;  differences:
```
hitbox.max_reach: 14.1
speed.enabled: false
noswing.enabled: false
log_file: false
aackick_broadcast: false
```
Transport to the first point works correctly and quickly.

En route to the second point, AAC reports:
```
[11:11:18 WARN]: playerName moved too quickly! -4446.7,0.0,6839.0
```
but after a significant delay, the player manages to get there. [NOTES: The real player name has been replaced above with playerName, to protect the guilty. The test player has OP permissions.]

En route to the third point, again after significant delay, the player loses connection to the server, and the server crashes.

[https://gist.github.com/SlimeDog/4f1ad885381117cac64a16952dfc1eb4](https://gist.github.com/SlimeDog/4f1ad885381117cac64a16952dfc1eb4)

**Comment by Janmm14** [comment]
The "moved too quickly" is not from AAC, its from vanilla

**Comment by Janmm14** [comment]
Could you do this:
1. Add AAC back to your server.
2. Use /aacdebug to toggle debug logging on.
3. Go to the second point.
4. Disable debug logging by using /aacdebug again.
5. Upload content of debug log file found in `plugins/AAC` to https://gist.github.com nad link it here.

**Comment by SlimeDog** [comment]
https://gist.github.com/SlimeDog/a3a3b3de54fc2cbd310cce9f7cb6f3d7

NOTE: The real player name has been replaced with _PLAYERNAME_.

**Comment by Janmm14** [comment]
I'm looking a bit puzzled on that debug log currently.

However in v3.6.3 something was changed regarding teleports which could maybe also fix this issue. It would be nice if you could do a quick test again with that version, maybe on a copy of the server and also create a debug log with that new version if the issue is not resolved.

Only problem with 3.6.3 is that it still contains debug messages printed to the console :/
But also the debug log contains different information on 3.6.3

**Comment by SlimeDog** [comment]
Spigot 1.13.2 latest (updated since above)
ProtocolLib 4.4.0-b422 (same as above)
AAC 3.6.3
ServerTutorialPlus 1.20.0  (same as above)

I am running all of this on test servers, so testing various configurations is not a problem.

[https://gist.github.com/SlimeDog/bfcf715eeae2132a4cd8dbd4e36c8242](https://gist.github.com/SlimeDog/bfcf715eeae2132a4cd8dbd4e36c8242)

NOTE: The real player name has been replaced with PLAYERNAME.

**Comment by SlimeDog** [comment]
No issues with AAC 3.6.4. I can traverse all the points.

It's OK (by me) to close this issue, but I see it marked **critical** a few minutes ago. Is there something further you would like me to test?

**Comment by Janmm14** [comment]
No, not really.

Thne this was morel ike accidentally fixed in 3.6.4
